.github/workflows/unit-test.yml

```diff
@@ -83,8 +83,17 @@ jobs:
         chmod -R 767 ./ # TODO: Possibly integrate in wp-env
         npm run wp-env start
 
-    - name: Running the tests
-      run: npm run test-php && npm run test-unit-php-multisite
```

Comment:
`test-php` just runs `lint-php && test-unit-php`, so we are just splitting it out into its own steps